### PR TITLE
Update vue generator example

### PR DESCRIPTION
**Pull request by cakriwut**
Yo generator vue is available in the office add ins generator.
- Add screenshot
- Use yo generator vue

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/yo-office-excel-vue.png](https://github.com/cakriwut/office-js-docs-pr/blob/update-yo-vue-article/docs/images/yo-office-excel-vue.png) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/cakriwut/office-js-docs-pr/blob/update-yo-vue-article/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809152239434619-373%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=E3%2Bk0eKcicEYrcVJANqQYD1zcZmYZ6tPM0QQpKaHHs4%3D&st=2018-09-15T22%3A36%3A08Z&se=2018-10-16T22%3A41%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/file-get-started-excel-vue.md [hunk 1]
```diff
@@ -16,131 +18,64 @@ In this article, you'll walk through the process of building an Excel add-in usi
...
-1. Navigate to your app folder.
+2. Navigate to your app folder.
 
     ```bash
-    cd my-add-in
+    cd my-addin
     ```
 
-2. Use the Yeoman generator to generate the manifest file for your add-in. Run the following command and then answer the prompts as shown below.
+3. Use the Yeoman generator to generate the manifest file for your add-in. Run the following command and then answer the prompts as shown in the following screenshot.
```
Comment (AlexJerabek) [inline docs/includes/file-get-started-excel-vue.md]: Can we change "as shown in the following screenshot" to "with the following choices"? We're describing the prompt answers in both text and the screenshot.

File: docs/includes/file-get-started-excel-vue.md [hunk 2]
```diff
@@ -16,131 +18,64 @@ In this article, you'll walk through the process of building an Excel add-in usi
...
-    const Office = window.Office
-    Office.initialize = () => {
-      new Vue({
-        el: '#app',
-        components: {App},
-        template: '<App/>'
-      })
-    }
-    ```
+    After you complete the wizard, the generator will create the project and install supporting Node components.
```
Comment (AlexJerabek) [inline docs/includes/file-get-started-excel-vue.md]: "the generator will create the project and install" -> "the generator creates the project and installs"

File: docs/includes/file-get-started-excel-vue.md [hunk 3]
```diff
@@ -16,131 +18,64 @@ In this article, you'll walk through the process of building an Excel add-in usi
...
-          window.Excel.run(async (context) => {
-            const range = context.workbook.getSelectedRange()
-            range.format.fill.color = 'green'
-            await context.sync()
-          })
-        }
-      }
-    }
-    </script>
+5.  Continue editing **src/App.vue**, in the `<script>` block, search for comment "Update the fill color." then change the fill color from 'yello' to 'blue' and save the file.
```
Comment (AlexJerabek) [inline docs/includes/file-get-started-excel-vue.md]: "yello" -> "yellow"

Comment (AlexJerabek) [reply docs/includes/file-get-started-excel-vue.md]: This sentence is kind of run-on. Consider splitting it into multiple sentences.

File: docs/includes/file-get-started-excel-vue.md [hunk 4]
```diff
@@ -200,7 +139,7 @@ To enable HTTPS for your app, open **package.json** in the root of the project,
 
 ## Next steps
 
-Congratulations, you've successfully created an Excel add-in using Vue! Next, learn more about the capabilities of an Excel add-in and build a more complex add-in by following along with the Excel add-in tutorial.
+Congratulations, you've successfully created an Excel add-in using Angular! Next, learn more about the capabilities of an Excel add-in and build a more complex add-in by following along with the Excel add-in tutorial.
```
Comment (AlexJerabek) [inline docs/includes/file-get-started-excel-vue.md]: "Angular" -> "Vue"

**Comment by cakriwut** [comment]
Hi @AlexJerabek , the code currently in pull request [Vue template \#352](https://github.com/OfficeDev/generator-office/pull/352)

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/yo-office-excel-vue.png](https://github.com/cakriwut/office-js-docs-pr/blob/update-yo-vue-article/docs/images/yo-office-excel-vue.png) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/cakriwut/office-js-docs-pr/blob/update-yo-vue-article/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809172310150973-373%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=EGIda83p%2FS7gb%2FBtN6snag2MNz6GWZbU08gYcB%2FJdnA%3D&st=2018-09-17T23%3A06%3A34Z&se=2018-10-18T23%3A11%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
@cakriwut Thanks for linking to the yo office PR. We can get this checked in after yo office is updated. We want the documentation to reflect the current state available to end-users.

**Comment by kbrandl** [comment]
@cakriwut I'm going to close this PR in the docs repo, since you've closed the related PR in OfficeDev/generator-office that these doc updates were dependent upon (https://github.com/OfficeDev/generator-office/pull/352). Thanks for your interest in Office Add-ins development!

**Pull request closed without merging**